### PR TITLE
[MOSTLY MODULAR] re-opens #11343

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request

gandalf is a silly goon and didn't look at the branch it was PR'd too

## How This Contributes To The Skyrat Roleplay Experience

see #11343, without the special ammo types.

## Changelog

:cl:
tweak: modularizes the WT550's as upstream removed them.
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -651,6 +651,8 @@
 					/obj/item/clothing/gloves/tackler/combat,
 					/obj/item/clothing/gloves/tackler/combat)
 	crate_name = "swat crate"
+	
+/datum/supply_pack/security/armory/woodstock // SKYRAT EDIT - ADDITION
```
Comment (Gandalf2k15) [inline code/modules/cargo/packs.dm]: ????????????

Comment (GoldenAlpharex) [reply code/modules/cargo/packs.dm]: lmao I think he forgot something there.

Comment (RatFromTheJungle) [reply code/modules/cargo/packs.dm]: it was overwritten in the aforementioned PR

Comment (RatFromTheJungle) [reply code/modules/cargo/packs.dm]: image

Comment (RatFromTheJungle) [reply code/modules/cargo/packs.dm]: i just assumed it was done on accident

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -651,7 +651,7 @@
 					/obj/item/clothing/gloves/tackler/combat,
 					/obj/item/clothing/gloves/tackler/combat)
 	crate_name = "swat crate"
-
+
```
Comment (Gandalf2k15) [inline code/modules/cargo/packs.dm]: ????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????

Comment (RatFromTheJungle) [reply code/modules/cargo/packs.dm]: ur making me very angry............

**Pull request merged**